diff --git a/node_modules/expo/node_modules/@expo/cli/.DS_Store b/node_modules/expo/node_modules/@expo/cli/.DS_Store
new file mode 100644
index 0000000..7d4a2ce
Binary files /dev/null and b/node_modules/expo/node_modules/@expo/cli/.DS_Store differ
diff --git a/node_modules/expo/node_modules/@expo/cli/build/.DS_Store b/node_modules/expo/node_modules/@expo/cli/build/.DS_Store
new file mode 100644
index 0000000..a907e0b
Binary files /dev/null and b/node_modules/expo/node_modules/@expo/cli/build/.DS_Store differ
diff --git a/node_modules/expo/node_modules/@expo/cli/build/src/.DS_Store b/node_modules/expo/node_modules/@expo/cli/build/src/.DS_Store
new file mode 100644
index 0000000..e6a44e9
Binary files /dev/null and b/node_modules/expo/node_modules/@expo/cli/build/src/.DS_Store differ
diff --git a/node_modules/expo/node_modules/@expo/cli/build/src/start/.DS_Store b/node_modules/expo/node_modules/@expo/cli/build/src/start/.DS_Store
new file mode 100644
index 0000000..4571886
Binary files /dev/null and b/node_modules/expo/node_modules/@expo/cli/build/src/start/.DS_Store differ
diff --git a/node_modules/expo/node_modules/@expo/cli/build/src/start/server/.DS_Store b/node_modules/expo/node_modules/@expo/cli/build/src/start/server/.DS_Store
new file mode 100644
index 0000000..77a371b
Binary files /dev/null and b/node_modules/expo/node_modules/@expo/cli/build/src/start/server/.DS_Store differ
diff --git a/node_modules/expo/node_modules/@expo/cli/build/src/start/server/middleware/CorsMiddleware.js b/node_modules/expo/node_modules/@expo/cli/build/src/start/server/middleware/CorsMiddleware.js
index 20c7bd5..3a82d02 100644
--- a/node_modules/expo/node_modules/@expo/cli/build/src/start/server/middleware/CorsMiddleware.js
+++ b/node_modules/expo/node_modules/@expo/cli/build/src/start/server/middleware/CorsMiddleware.js
@@ -17,7 +17,8 @@ _export(exports, {
     }
 });
 const DEFAULT_ALLOWED_CORS_HOSTS = [
-    'devtools'
+    'devtools',
+    'tauri.localhost'
 ];
 const _isLocalHostname = (hostname)=>{
     if (hostname === 'localhost') {
